brave/browser-laptop
# Added default download path

Resolves #2110

## Auditors
@bradleyrichter @bsclifton

## Test Plan
- go to the preferences, general tab
- define your default download path
- toggle show dialog
- based on your settings, download dialog should be opened or not
- if you set to hide download dialog, files should be downloaded automatically to your default download path
